samples/bluetooth/hci_uart/nrf52840_pca10090.conf

```diff
@@ -0,0 +1,26 @@
+CONFIG_GPIO=y
+CONFIG_SERIAL=y
+CONFIG_MAIN_STACK_SIZE=512
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=512
+CONFIG_BT=y
+CONFIG_BT_HCI_RAW=y
+CONFIG_BT_MAX_CONN=16
+CONFIG_BT_TINYCRYPT_ECC=n
+CONFIG_BT_CTLR_DTM_HCI=y
+CONFIG_BT_CTLR_ASSERT_HANDLER=y
+
+# We can spare a UART for the console
+CONFIG_CONSOLE=y
+CONFIG_STDOUT_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# Connect the MCU interface pins to the nRF9160
+# We will instantiate UART1 on these pins
+CONFIG_BOARD_PCA10090_INTERFACE_0_MCU=y
+CONFIG_BOARD_PCA10090_INTERFACE_1_MCU=y
+
+# Use UART1 for HCI
+CONFIG_UART_1_NRF_UARTE=y
+CONFIG_UART_1_NRF_FLOW_CONTROL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_BT_CTLR_TO_HOST_UART_DEV_NAME="UART_1"
```

Comment:
Is this necessary, since this is part of the `.overlay`?

Reply:
I double checked: `CONFIG_BT_CTLR_TO_HOST_UART_DEV_NAME` will default to "UART_0" regardless of which UART is specified in the `dts`.